flyylikeaduck/js-test
# Created simple for loop

## What it does
 - creates simple for loop
 - comments out unneeded code

## How to test
 - run in console
 - see "woof" print out 3x

## Notes
 - This is my first pull request!! 👯
